satellite/peer.go

```diff
@@ -417,8 +407,10 @@ func (peer *Peer) Run(ctx context.Context) error {
 	})
 	group.Go(func() error {
 		// TODO: move the message into Server instead
-		peer.Log.Sugar().Infof("Node %s started on %s", peer.Identity.ID, peer.Public.Server.Addr().String())
-		return ignoreCancel(peer.Public.Server.Run(ctx))
+		// Don't change the format of this comment, it is used to figure out the node id.
+		peer.Log.Sugar().Infof("Node %s started on %s", peer.Identity.ID, peer.Addr())
+		peer.Log.Sugar().Infof("Node %s started on %s", peer.Identity.ID, peer.PrivateAddr())
```

Comment:
maybe this log message could distinguish private vs public

Reply:
good idea. I will add public vs private